crypto/fipsmodule/rsa/internal.h

```diff
@@ -112,6 +111,22 @@ int RSA_padding_add_none(uint8_t *to, size_t to_len, const uint8_t *from,
 // within DoS bounds.
 int rsa_check_public_key(const RSA *rsa);
 
+// An enum used internally to designate what type of RSA key we detect upon
+// validation. Used to determine which validation rules to enforce.
+typedef enum {
+    RSA_KEY_PRIVATE_STRIPPED,
```

Comment:
"Stripped keys" does not have a well-defined semantic (AFAIK).
We should be clear why we introduce this classification and where it comes from.

Reply:
we discussed this offline and arrived at the consensus that this change should keep its scope as narrow as possible while still supporting keys passed from JCA/ACCP. the net effect of that the mos recent revision ([`8acfe39`](https://github.com/awslabs/aws-lc/commit/8acfe39fe5059a65414d795c50f4c9ff7d9e2182)) hews much more closely to torben's [initial sketch](https://github.com/torben-hansen/aws-lc/compare/main...thoughts_rsa_accp_key_check), with a few modifications by me to get it to compile, account for some ACCP quirks, add test coverage, etc.